tests/components/unifi/test_update.py

```diff
@@ -158,16 +162,13 @@ async def test_not_admin(hass, aioclient_mock):
 
 async def test_install(hass, aioclient_mock):
     """Test the device update install call."""
-    config_entry = await setup_unifi_integration(
-        hass, aioclient_mock, devices_response=[DEVICE_1]
-    )
+    await setup_unifi_integration(hass, aioclient_mock, devices_response=[DEVICE_1])
 
     assert len(hass.states.async_entity_ids(UPDATE_DOMAIN)) == 1
     device_state = hass.states.get("update.device_1")
     assert device_state.state == STATE_ON
 
-    controller = hass.data[UNIFI_DOMAIN][config_entry.entry_id]
-    url = f"https://{controller.host}:1234/api/s/{controller.site}/cmd/devmgr"
+    url = f"https://{DEFAULT_HOST}:1234/api/s/{DEFAULT_SITE}/cmd/devmgr"
```

Comment:
We can access the config entry data to get the host and site of this controller.

https://github.com/home-assistant/core/blob/ba8a530d19dc3e93602275961de46d700611f4a1/homeassistant/components/unifi/controller.py#L166-L174